app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, copied, file_to_copy = command.split()
+    if copied != file_to_copy or _ == "cp":
```

Comment:
use underscore for varuable only if u won't use this variable later, u are using it in if condition